app/assets/stylesheets/patternfly_and_overrides.scss

```diff
@@ -315,3 +315,7 @@ a.btn-info span{
   max-height: 80vh;
   overflow-y: auto;
 }
+
+.pficon-ok.warn:before {
+  color: #ec7a08;
```

Comment:
nitpick: I would add a comment this is pf-orange-400 color, just in case we ever search for it.